### ISSUE TITLE
gpu.bind(ID, false) unreasonably slow

### ISSUE THREAD

**Issue by Nex4rius** [MightyPirates/OpenComputers#3062]
```lua
gpu.bind(ID)
gpu.setResolution(X, Y)
```
This switched the GPU to the new screen and changes the resolution to the maximum resolution and after that it changes the resolution to X, Y.
1) new screen
2) resolution change
3) resolution change
```lua
gpu.bind(ID, false)
```
This switches the GPU to a new screen without changing anything.
1) new screen

<hr>

Why is the second method 3 times slower than the first even though it does only 1 thing instead of 3 things?
I don't think it makes a lot of sense for it to be slower.

This is really an issue if you have lots of screens and don't want to use the faster method because having multiple unnecessary resolution changes looks like flickering on the screens.

I tested it with <a href="https://gist.github.com/Nex4rius/63d1e53616ea4abfe0cedaf8ec9ea80d#file-test-lua">this</a> and made a <a href="https://www.youtube.com/watch?v=fDKXh0o11qs&feature=youtu.be">video of it</a>.

OpenComputers-MC1.12.2-1.7.4.153

**Comment by anar4732** [comment]
I agree. this is a very frustrating problem.

**Comment by skyem123** [comment]
I think this was a deliberate decision. I cannot seem to find an issue where this was discussed, but if I recall correctly, the idea was to discourage people from using a single GPU to frequently update multiple monitors? I do not know if that is a good idea in hindsight though.

**Comment by payonel** [comment]
I want these two gpu.bind calls to take roughly the same amount of time. I have verified what you have reported, and after a TON of testing I believe the implementation did not produce the effects we intended with this time cost, and that a simple change can get us safely to the experience we want.

Basically, we wanted there to be a cost in using a single gpu to control multiple screens. If you wanted fast updates to multiple screens, you would benefit from having multiple gpus. The intention here was that you would manage each screen with its own gpu for optimal performance.

To help demonstrate this, I made a variation of this test script here: https://gist.github.com/payonel/04e02166c5562ca49aba219993d8c7aa

My version splits of the screens to the available gpus, moves the color index by only 10, and keeps a cache of the screen ids (there is a time cost in iterating component.list)

With a single gpu, Updating a single screen takes ~1s. For any number of screens greater than 1, the test takes ~25s per screen.

With two gpus and 1 or 2 screens, the test takes ~1s. For any gpu that manages more than 1 screen, each screen costs ~25s.
